Specs/Scene/ModelExperimental/ModelColorPipelineStageSpec.js

```diff
@@ -103,7 +102,7 @@ describe("Scene/ModelExperimental/ModelColorPipelineStage", function () {
       uniformMap: {},
       model: mockModel,
       alphaOptions: defaultAlphaOptions,
-      renderStateOptions: {},
+      renderStateOptions: RenderState.getState(RenderState.fromCache()),
```

Comment:
Similarly here, could we create a `mockRenderResources()` function for this file?